src/public/modules/forms/admin/directives/settings-form.client.directive.js

```diff
@@ -156,15 +154,22 @@ function settingsFormDirective(
           {
             val: 'NIL',
             name: 'None',
+            isEnabledInStorageMode: true,
```

Comment:
would it be simpler to avoid using `isEnabledInStorageMode` and rely on say, `isFormEncrypt()` instead?

Reply:
I don't think so actually. there are two conditions we need to check: whether the form is storage mode, and whether the given auth type is allowed in storage mode. we need `isFormEncrypt` for the first and `isEnabledInStorageMode` for the second.